### PR TITLE
feat: [PL-43630]: Implemented AbsoluteSessionTimeout

### PR DESCRIPTION
**Pull request by mayankVermaHarness**
### Summary

JIRA: https://harness.atlassian.net/browse/PL-43630
Implemented AbsoluteSessionTimeout

Added a new field for AbsoluteSessionTimeout along with Session Inactivity Timeout.
Default value for AbsoluteSessionTimeout is 0 (which means it's not set). 
When AbsoluteSessionTimeout is set, user will get logged out despite performing `Activity`—`Activity` is defined as a `mouse-click` event or a keyboard `key-press` event.

#### Screenshots

image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

**Review by namanharness — APPROVED** [review]

File: src/modules/27-platform/auth-settings/pages/Configuration/SessionTimeOut/SessionTimeOut.tsx [hunk 1]
```diff
@@ -101,43 +136,83 @@ const SessionTimeOut: React.FC<SessionTimeOutProps> = ({ timeout }) => {
...
+                      <Layout.Vertical spacing="small">
+                        <Layout.Horizontal spacing="xxxlarge" flex={{ justifyContent: 'flex-start' }}>
+                          <TextInput
+                            type="number"
+                            value={formik.values.absoluteSessionTimeout as any}
+                            min={MINIMUM_ABSOLUTE_SESSION_TIMEOUT_IN_MINUTES}
+                            max={MAXIMUM_ABSOLUTE_SESSION_TIMEOUT_IN_MINUTES}
+                            wrapperClassName={css.textInpt}
+                            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                              formik.setFieldValue('absoluteSessionTimeout', e.target.value)
```
Comment (abhinavrastogi-harness) [inline src/modules/27-platform/auth-settings/pages/Configuration/SessionTimeOut/SessionTimeOut.tsx:190]: why is this needed?

Comment (abhinavrastogi-harness) [reply src/modules/27-platform/auth-settings/pages/Configuration/SessionTimeOut/SessionTimeOut.tsx:190]: using `defaultValue` instead of `value` will remove the need for `onChange`

**Comment by abhinavrastogi-harness** [comment]
kudos on removing the snapshots

File: src/modules/27-platform/auth-settings/pages/Configuration/SessionTimeOut/SessionTimeOut.tsx [hunk 2]
```diff
@@ -101,43 +136,83 @@ const SessionTimeOut: React.FC<SessionTimeOutProps> = ({ timeout }) => {
...
+                            }
+                            name="absoluteSessionTimeout"
+                          />
+                          <Button
+                            type="submit"
+                            text={getString('save')}
```
Comment (sunnykesh-harness) [inline src/modules/27-platform/auth-settings/pages/Configuration/SessionTimeOut/SessionTimeOut.tsx:196]: @mayankVermaHarness This Save button is common to both settings but right now aligned in right with last one. This can be a little confusing compared with when it is placed at bottom.
